frontend/hub/administration/remotes/RemotePage/RemoteUserAccess.tsx

```diff
@@ -0,0 +1,39 @@
+import { useParams } from 'react-router-dom';
+import { LoadingPage } from '../../../../../framework';
+import { UserAccess } from '../../../../common/access/components/UserAccess';
+import { useGet } from '../../../../common/crud/useGet';
+import { pulpAPI } from '../../../common/api/formatPath';
+import { HubError } from '../../../common/HubError';
+import { PulpItemsResponse } from '../../../common/useHubView';
+import { HubRoute } from '../../../main/HubRoutes';
+import { HubRemote } from '../Remotes';
+
+export function RemoteUserAccess() {
+  const params = useParams<{ id: string }>();
+
+  const { data, error, refresh } = useGet<PulpItemsResponse<HubRemote>>(
+    pulpAPI`/remotes/ansible/collection/?name=${params.id}`
+  );
+
+  let remote: HubRemote | undefined = undefined;
+  if (data && data.results && data.results.length > 0) {
+    remote = data.results[0];
+  }
+
+  if (!data && !error) {
+    return <LoadingPage />;
+  }
+
+  if (error) {
+    return <HubError error={error} handleRefresh={refresh} />;
+  }
+
+  return (
+    <UserAccess
+      service="hub"
+      id={remote?.name?.toString() || ''}
+      type={'remote'}
```

Comment:
```suggestion
      type={'collectionremote'}
```
This is the content-type for remotes and will be used in the content_type__model filter.